library/swarm

```diff
@@ -1,10 +1,9 @@
-# maintainer: Alexandre Beslic <abronan@docker.com> (@abronan)
-# maintainer: Andrea Luzzardi <al@docker.com> (@aluzzardi)
-# maintainer: Dong Chen<dong@docker.com> (@dongluochen)
-# maintainer: Nishant Totla<nishanttotla@gmail.com> (@nishanttotla)
-# maintainer: Victor Vieux <vieux@docker.com> (@vieux)
+Maintainers: Alexandre Beslic <abronan@docker.com> (@abronan),
+             Andrea Luzzardi <al@docker.com> (@aluzzardi),
+             Dong Chen <dong@docker.com> (@dongluochen),
+             Nishant Totla <nishanttotla@gmail.com> (@nishanttotla),
+             Victor Vieux <vieux@docker.com> (@vieux)
+GitRepo: https://github.com/docker/swarm-library-image.git
 
-
-1.2.4: git://github.com/docker/swarm-library-image@ad016670d877529ab32e7ef2a1323777a06cb46a
-1.2.5: git://github.com/docker/swarm-library-image@67e6cf4171fcabd2e3b8b809105e7dfb15114ce1
-latest: git://github.com/docker/swarm-library-image@67e6cf4171fcabd2e3b8b809105e7dfb15114ce1
+Tags: 1.2.5, latest
+GitCommit: 67e6cf4171fcabd2e3b8b809105e7dfb15114ce1
```

Comment:
Just add a new section; something like this should do it:
```
Maintainers: ...
GitRepo: ...

Tags: 1.2.5, latest
GitCommit: 67e6cf4171fcabd2e3b8b809105e7dfb15114ce1

Tags: 1.2.6-rc1
GitCommit: xxxxxxx
```

Then, once 1.2.6 officially releases, you remove the second entry and update the first again. :+1: